tests/test_crp_strategy.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from math import isclose
+from typing import *
+from olps.strategies.strategy.crp_strategy import CRPStrategy
```

Comment:
`.strategy` should not be here. I understand it makes sense from an OOP perspective, but think of how the files are laid out instead